src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from './Loader';
+import { TablePage } from './TablePage';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsLoadingError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading ? (
+            <Loader />
```

Comment:
```suggestion
          {isLoading && <Loader />}
```

Your nesting looks to complicated, we can refactor it in this way